ucm2/AMD/acp3xalc5682m98/HiFi/2mic.conf

```diff
@@ -0,0 +1,108 @@
+SectionVerb {
```

Comment:
It seems that there are minimal changes in the configs. It would be probably much better to have one HiFi.conf configuration with If blocks to cover differences for this type of hw to simplify the future maintenance.